tools/github/Add-AzsdkMilestones.ps1

```diff
@@ -1,5 +1,9 @@
-[CmdletBinding(DefaultParameterSetName = 'Repositories', SupportsShouldProcess = $true)]
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
 param (
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string]$RepositoryFilePath = "./repositories.txt",
```

Comment:
Is this truly intended to use the current working directory i.e., `$PWD`, or the directory the script is in? If the former, great. If the latter, you'll need to use `$PSScriptRoot/repositories.txt`.

Reply:
The intent was the former.   Do you think the latter makes more sense?

Reply:
Reading the feedback further down has me reconsidering.  I'll adjust to script root.

Reply:
You'll also need to use double quotes as you are. Variables aren't resolved in single quotes, which I recommend using by default to avoid accidental variable resolution, but that's more of a preference/tip.